BaseTools/Source/C/Makefiles/header.makefile

```diff
@@ -54,13 +54,11 @@ AS ?= $(CLANG_BIN)clang
 AR ?= $(CLANG_BIN)llvm-ar
 LD ?= $(CLANG_BIN)llvm-ld
 else ifeq ($(origin CC),default)
-# MU_CHANGE STARTs: Support GCC prefix
-CC ?= $(GCC_PREFIX)gcc
-CXX ?= $(GCC_PREFIX)g++
-AS ?= $(GCC_PREFIX)gcc
-AR ?= $(GCC_PREFIX)ar
-LD ?= $(GCC_PREFIX)ld
-# MU_CHANGE ENDs
+CC = $(GCC_PREFIX)gcc

```

Comment:
I assume switching these to unconditional assignments was intentional?

Reply:
Yeah. This is intentionally added because CC was set to the host based gcc during the process and we need it as a cross compiler for this step.